### DISCUSSION TITLE
Stop Adonis js server (v4)

### DISCUSSION THREAD

**Discussion by purnimadas27** [adonisjs/core#1811] [Help] [answered]
Hello,

I have been struggling to stop my Adonis js server (v4), can someone help? 

My Adonis JS server has been install on linode (Ubuntu) environment (it's our UAT env.), and when I failed to find any stop command, I tried killing the process using pid, however, for some reason, if I kill my node process, immediately a new one starts with a different PID, I was wondering if similar to the 'adonis serve --dev --debug' start server command there is any stop command available... ? 

Obviously I am missing something here, can someone provide some guidance ?

Thank you,
Purnima Das

**Comment by hariprasath-yadav** [comment, marked as answer]
I can help you, but if you willing to share your screen via anydesk

**Reply by purnimadas27** [reply]
Hello,

Thank you, but I finally was able to figure it out, let me provide the details here for those who may find this useful:

We used PM2 or Process Manager to start / stop our node js server, and you will be able to access your logs under ../.pm2/logs folder, PM2 also provides commands to start / stop / restart / etc your node js server. Visit the following site for more details:
https://pm2.keymetrics.io/docs/usage/quick-start/

Thank you,
Purnima Das

**Reply by hariprasath-yadav** [reply]
I'm glad you find the solution with PM2 package.
